README.md

```diff
@@ -1,3 +1,65 @@
 # Python version
 
 I am using python version 3.11 and above
+
+# Project's Title
+
+FastAPI App
+
+# Description
+
+This is simple web application that allows create and get data from database
+
+# Technologies used
+
+FastAPI - A framework for rapid API development with asynchronous support and Swagger/OpenAPI automatic documentation.
+Python - Programming language
+Uvicorn - ASGI server used to run FastAPI applications.
+Git - A version control system for code collaboration
+HTTP - The project is built on the principles of the HTTP protocol
+
+# Install 
+
+1.Clone the repository
+
+git clone https://github.com/Katerrynaa/Tasks.git
+
+2.Navigate to the project directory
+
+cd test/
+
+3.Create virtual environment
+
+python -m venv venv
+
+4.Activate virtual environment
+
+source .venv/bin/activate
+
+5.Install requirements
+
+pip install -r requirements.txt
+
+# Run 
+
+1.Run fastapi server
+
+uvicorn main:app --reload
+
+2.Open your browser and go to the address
+
+http://127.0.0.1:8000/docs
+
+# Tests
+
+Project has tests to check functionality API. Use this command to run the tests:
+
+make test
+
+# License 
+
+This project is licensed under [MIT License](C:\Users\katad\Desktop\python\test\LICENSE.md)
```

Comment:
Not a good idea to put here link to your local file system)) It does not work even for you now, after linux installation)